src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -1093,86 +1099,82 @@ export function getAlertContent(alert) {
       };
     }
     case ALERT_TYPES.scheduledHearing: {
-      const formattedDate = moment(alert.date, 'YYYY-MM-DD').format('MMMM DD, YYYY');
       return {
-        title: `Your hearing is scheduled for ${formattedDate}`,
-        description: null,
+        title: (
+          <span>A hearing has been scheduled. <a href="/disability-benefits/claims-appeal/hearings/">Learn more about hearings, including how to prepare for, reschedule, or cancel your hearing</a>.</span>
+        ),
+        // TODO: determine whether details.type or type should be used here
```

Comment:
[This](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/9095) issue recommends using the details object in the alert content, to what extent should property access emphasize using `details`? Are there ways in which that can be increased here?

Reply:
Sorry, ya'll. :bashful:

Reply:
Thanks @cvalarida @cmgiven.